crates/wash-lib/src/build/component.rs

```diff
@@ -309,8 +310,21 @@ async fn build_tinygo_component(
             component_config.wit_world.as_ref().context(
                 "missing `wit_world` in wasmcloud.toml ([component] section) to run go bindgen generate",
             )?,
-            ".",
-        ],
+        ];
+
+            if let Some(scheduler) = &tinygo_config.scheduler {
+                args.push("-scheduler");
+                args.push(scheduler);
+            }
+
+            if let Some(gc) = &tinygo_config.garbage_collector {
+                args.push("-gc");
+                args.push(gc);
+            }
+
+            args.push(".");
+            args
```

Comment:
Is the `.` a new requirement? I don't see the `.` in my tinygo build commands, but perhaps this is required after the other flags?

Reply:
not a new requirement ( it's also in the wasip1 target ).
it's a package path selector and not needed when the package you want to build is the one in `go.mod` ( most cases when  building components ).
It does matter when building components in monorepos.